api/group.py

```diff
@@ -213,3 +213,38 @@ def delete_hosts_from_group(group_id, host_id_list, rbac_filter=None):
         abort(status.HTTP_404_NOT_FOUND, "Group or hosts not found.")
 
     return Response(None, status.HTTP_204_NO_CONTENT)
+
+
+@api_operation
+@rbac(RbacResourceType.GROUPS, RbacPermission.WRITE)
+@metrics.api_request_time.time()
+def delete_hosts_from_different_groups(host_id_list, rbac_filter=None):
+    check_inventory_groups_flag()
+
+    hosts_per_group = {}
+
+    # Separate hosts per group
+    for host_id in host_id_list:
+        group = get_group_from_host_id(host_id)
+
+        if group and group.id not in hosts_per_group:
+            hosts_per_group[str(group.id)] = [host_id]
+        elif group and group.id in hosts_per_group:
+            hosts_per_group[str(group.id)].append(host_id)
+
+    requested_group_ids = set(hosts_per_group.keys())
+
+    rbac_group_id_check(rbac_filter, requested_group_ids)
+
+    delete_count = 0
+
+    for group_id in requested_group_ids:
+        deleted_for_group = remove_hosts_from_group(
```

Comment:
Should this variable be named `deleted_from_group`?
If not, forget it!!

```suggestion
        deleted_from_group = remove_hosts_from_group(
```

Reply:
that's right!